egs/librispeech/ASR/pruned_transducer_stateless2/optim.py

```diff
@@ -287,7 +287,8 @@ def get_lr(self):
         factor = (
             (self.batch**2 + self.lr_batches**2) / self.lr_batches**2
         ) ** -0.25 * (
-            ((self.epoch**2 + self.lr_epochs**2) / self.lr_epochs**2) ** -0.25
```

Comment:
@AmirHussein96 could you remove these formatting changes? Please see https://github.com/k2-fsa/icefall/pull/692 where we updated the line-length.

Reply:
I finished the formatting

Reply:
Thanks.